gdnative-derive/src/native_script.rs

```diff
@@ -10,12 +12,25 @@ use property_args::{PropertyAttrArgs, PropertyAttrArgsBuilder};
 pub(crate) struct DeriveData {
     pub(crate) name: Ident,
     pub(crate) base: Type,
-    pub(crate) register_callback: Option<Path>,
+    pub(crate) register_callback: Option<Vec<Path>>,
```

Comment:
Does it make sense to make the distinction between the case of having an empty `register_with` attribute, and the case of not having one at all? If not, this can simply be `Vec<Path>` instead.